bundle/regal/rules/idiomatic/circular_import_test.rego

```diff
@@ -0,0 +1,265 @@
+package regal.rules.idiomatic["circular-import_test"]
```

Comment:
I suppose avoiding this is idomatic, but I wonder if the `imports` category would be even more appropriate?

Reply:
yeah I can move it to there!

Reply:
Moved in https://github.com/StyraInc/regal/pull/558/commits/c628729de019281fab625f49ebcf0b3e67294db9